src/App.jsx

```diff
@@ -19,9 +19,9 @@ export default function App() {
             <Route path="/login">
               <Login />
             </Route>
-            <Route path="/treehouse/members">
```

Comment:
Great job here updating your routes to use your context provider which makes the user state accessible to the entire app.